xarray/test/test_variable.py

```diff
@@ -213,7 +213,10 @@ def test_pandas_period_index(self):
     def test_1d_math(self):
         x = 1.0 * np.arange(5)
         y = np.ones(5)
-        v = self.cls(['x'], x)
+
+        # should we need `.to_base_variable()`?
+        # probably a break that `+v` changes type?
+        v = self.cls(['x'], x).to_base_variable()
```

Comment:
Instead of always converting `v` to a base variable, I would make another copy that is the base variable and using that for the expected variable. Thus these tests will still check that math with `IndexVariable` works even though it doesn't preserve the type.